views/jobRoles.njk

```diff
@@ -13,9 +13,26 @@
         <div class="container">
 
             <div class="mainBody">
-                <h1>Job Roles List</h1>
+                <h1 class = "m-0 p-0">Job Roles List</h1>
+                <div class = "d-flex flex-row justify-content-between m-0 p-0">
```

Comment:
I want a line gap between the Job Roles List title and the pagination total at the top of the page and the drop down should be inline with the pagination total after this change.

Reply:
Added